### PR TITLE
Tidy-up

### PR DESCRIPTION
**Pull request by mikenye**
Remove a superfluous "TEXT1".

### CODE REVIEW

**Comment by mikenye** [comment]
All checks pass now.

File: .spelling [hunk 1]
```diff
@@ -89,6 +104,7 @@ mlat_port
 mlat_url
 mlat.planespotters.net
 Multilateration
+ommercial
```
Comment (fredclausen) [inline .spelling:107]: I’m on my phone and can’t do a proper search, but this seems like an odd word

Comment (fredclausen) [reply .spelling:107]: Nvm I found it. Disregard :)

**Review by fredclausen — APPROVED** [review]
Looks good!

**Review by kx1t — APPROVED** [review]
Approved; however I am going to revert this PR to "draft" so we can make additional changes to https://github.com/sdr-enthusiasts/gitbook-adsb-guide/blob/1d5f7c183e0ccc7a50a6140659b79d13298996cd/useful-extras/alternative-graphing-with-prometheus-grafana.md

**Pull request merged**